app/routes/upload.step2.tsx

```diff
@@ -15,24 +16,36 @@ export const action = async ({ request }: ActionFunctionArgs) => {
 
```

Comment:
**nitpick**: Remove commented-out code to keep the codebase clean and maintainable.